eng/scripts/DownloadDevopsLogger.ps1

```diff
@@ -0,0 +1,6 @@
+New-Item -ItemType directory artifacts -Force
+Invoke-WebRequest -Uri https://vstsagenttools.blob.core.windows.net/tools/msbuildlogger/4/msbuildlogger.zip -OutFile artifacts/msbuildlogger.zip
```

Comment:
I'm not a big fan of this method of tool acquisition. Do you know if they have a nuget package or something we could reference instead?

Reply:
Nope, I don't like it too but it seems that's how everyone does it : https://grep.app/search?q=https%3A//vstsagenttools.blob.core.windows.net/tools/msbuildlogger

Reply:
I wonder if it is already on the agents if so perhaps we don't have to download it only create a reference to it.

Reply:
Not that that's a very good argument :)

Reply:
I can check the file in.

Reply:
I wouldn't commit the zip file, but we should see if it is already on the machine and if so we should use that one.

Reply:
If it was on the machine I would suspect the dotnet azure pipelines task to be using it (https://github.com/microsoft/azure-pipelines-tasks/pull/15496) instead of carrying their own.

Reply:
I guess in my mind I was thinking that if they are already handling this, we could just do some dummy step like outputting the version or something that would cause the logger to be downloaded and managed by the task instead of us needing to maintain that code.